frontend/catalyst/jax_primitives.py

```diff
@@ -2333,6 +2387,15 @@ def _cos_lowering2(ctx, x, accuracy):
     return _nary_lower_hlo(hlo.cosine, ctx, x, accuracy=accuracy)
 
 
+def subroutine_lowering(*args, **kwargs):
+    """This is just a method that forwards arguments to _pjit_lowering
+
+    Useful for testing.
```

Comment:
This says it's for testing, but isn't this the lowering you actually registered? Then it's used in production right?

Reply:
```suggestion
    Even though we could register the `pjit_p` lowering directly, this makes the code origin
    apparent in stack traces and similar use cases.
```